pkg/cli/interface.go

```diff
@@ -17,14 +17,22 @@ package cli
 import (
 	"github.com/jonboulle/clockwork"
 	"github.com/tektoncd/pipeline/pkg/client/clientset/versioned"
+	k8s "k8s.io/client-go/kubernetes"
 )
 
+type Clients struct {
+	Tekton versioned.Interface
+	Kube   k8s.Interface
+}
+
 // Params interface provides
 type Params interface {
 	// SetKubeConfigPath uses the kubeconfig path to instantiate clientset
 	// returned by Clientset function
 	SetKubeConfigPath(string)
 	Clientset() (versioned.Interface, error)
```

Comment:
"there can be only one" :sweat_smile: As `Clients` holds both `versions.Interface` and `k8s.Interface`, `Clientset()` and `KubeClient()` should go away :angel:

Reply:
yeah, I was also wondering. just want to confirm, is there any case where it would be good to create k8s client only? otherwise, let me make those changes

Reply:
I don’t think there is any case :) and if it turns out there is, we can think about it then :wink:

Reply:
Agree it is better to Clientset of return `*Client` which holds all clients.

Reply:
resolved :bowing_man: , please confirm if its the right

Reply:
resolving this, please free to open it again :sweat_smile: